configure.ac

```diff
@@ -10,6 +10,7 @@ dnl older automake's default of ARFLAGS=cru is noisy on newer binutils;
 dnl we don't really need the 'u' even in older toolchains.  Then there is
 dnl older libtool, which spelled it AR_FLAGS
 m4_divert_text([DEFAULTS], [: "${ARFLAGS=cr} ${AR_FLAGS=cr}"])
+m4_include(m4/ax_compare_version.m4)
```

Comment:
Please quote this, so `m4_include([m4/ax_compare_version.m4])`.
Can you also add a newline above it to make it clear that this code has nothing to do with libtool?